### PR TITLE
Add extra links for Instagram

### PR DESCRIPTION
**Pull request by CarrotManMatt**
After the failure to deploy last night (#501), this branch needs to be remerged.

### CODE REVIEW

**Comment by CharlesESW** [comment]
Yeah buddy I ain't got a clue, I checked the logs in each fail and it says Command failed with exit code 1: jekyll build (https://ntl.fyi/exit-code-1) but can't find a solution online. My review of this pull request is that it all looks ok but evidently netlify is having trouble building. I don't know much about netlify.

**Comment by Thatsmusic99** [comment]
I'll give a look when I get home

File: _data/links.yml [hunk 1]
```diff
@@ -2,6 +2,26 @@
...
+
+- text: "Find out about AFNOM's 'Missing Semester'"
+  url: https://missingsemester.afnom.net/
+  image: /assets/images/afnom_logo.png
+
+- text: Buy a CSS rubber duck
+  url: /ducks
+  image: https://www.guildofstudents.com/asset/Event/6531/CSSDuck.png
+
+- text: Meet our mascot: TeX
```
Comment (Thatsmusic99) [inline _data/links.yml:21]: Syntax error found here - should be wrapped in quotes (`- text: 'Meet our mascot: Tex'`, or alternatively `- text: Meet our mascot, Tex!`)

**Review by Thatsmusic99 — CHANGES_REQUESTED** [review]

File: _data/links.yml (hunk 1 above)
Comment (CarrotManMatt) [reply _data/links.yml:21]: Awesome! Thank u so much for doing the full check for this, I just found the build logs unreadable

Comment (Thatsmusic99) [reply _data/links.yml]: Just for convenience this was how I figured it out, it's not very clear but once you find it:

image

**Review by Thatsmusic99 — CHANGES_REQUESTED** [review]
Two more things now that it compiles :)

- The facebook image seems to be missing
- The text also overflows

image

Do we also have the links page... linked anywhere?

**Comment by CarrotManMatt** [comment]
> * The facebook image seems to be missing
I am an excellent programmer referencing files that don't exist

**Comment by Thatsmusic99** [comment]
Schrodinger's image

**Comment by CarrotManMatt** [comment]
> Do we also have the links page... linked anywhere?

Should be in the Instagram Bio & well as the Discord TeXBot's bio

**Comment by CarrotManMatt** [comment]
> * The text also overflows

Annoyingly the scaling is messed up so on smaller screens some of the links will overflow as well, but I'll reduce this one to prevent it in the majority of cases

**Comment by Thatsmusic99** [comment]
tbh if Instagram is the main platform that this will be accessed through, people are gonna be accessing with smaller screens, so it may be worth tackling this head-on, I'll be happy to take a look once I've finished my CAV assignment

**Comment by CarrotManMatt** [comment]
awesome! thank you!

**Comment by Thatsmusic99** [comment]
Changes for overflowing text made with extra image padding, please can other committee help review thanks <3

**Comment by codemicro** [comment]
May I suggest we leave out Facebook? Just because our announcements _don't_ go on Facebook and I feel like that's leading people a little astray.

**Comment by MattyTheHacker** [comment]
I think we could start moving away from Facebook but for the timebeing I think since we do still have the group and the redirects we should leave it in.

**Review by MattyTheHacker — APPROVED** [review]

**Review by MattyTheHacker — APPROVED** [review]

**Review by Thatsmusic99 — APPROVED** [review]

**Pull request merged**